eversql/README.md

```diff
@@ -0,0 +1,41 @@
+# EverSQL
+
+## Overview
+
+[EverSQL][1] is the fastest way to automatically speed up your database and optimize SQL queries, providing automatic SQL tuning and indexing for developers, DBAs, and DevOps engineers.
+
+EverSQL is 100% non-intrusive, and doesn't access any of your databases' sensitive data.
+
+### Usage
+
+Slow SQL queries located in the Datadog Database monitoring dashboard can be optimized instantly using EverSQL, by easily copying the slow SQL query from Datadog and pasting it directly into EverSQL's [SQL Optimization][2] process.
+
+### Supported Databases: 
+MySQL, PostgreSQL, AWS Aurora, Google Cloud SQL, Azure DB, Percona, MariaDB.
+
+## Setup
+
+### Configuration
+To speed up slow queries identified by Datadog, navigate to the Datadog Database monitoring dashboard and locate the slow SQL queries table. Once you identify the SQL query you'd like to speed up, copy it from Datadog and paste it directly into [EverSQL][2] for optimization.
```

Comment:
```suggestion
To speed up slow queries identified by Datadog, navigate to the Datadog Database Monitoring dashboard and locate the slow SQL queries table. Once you identify the SQL query you'd like to speed up, copy it from Datadog and paste it directly into [EverSQL][2] for optimization.
```
Maybe it would be helpful to add a link to the Database Monitoring dashboard here?